### DISCUSSION TITLE
frontend web page loads, but "Loading please wait" message remains

### DISCUSSION THREAD

**Discussion by forestial** [Koenkk/zigbee2mqtt#10235] [General]
I installed zigbee2mqtt on a Raspberry PI and added several devices.
I enabled the frontend using port 8090
I started zigbee2mqqt (interactively, not yet using it as a service)

I try to load the frontend webpage in a browser.  The static parts of the page load (including the menu bar at the top of the page).  However the message "Loading please wait" is displayed and it never goes away.  
Have tried with several browsers (Firefox, Edge from a PC; Chromium from the PI itself).  Same result everywhere.

**Comment by mausch** [comment]
If you open the dev console on Chrome (F12), what do you see in the console/network tabs? Any http errors in there?
Also how are you running zigbee2mqtt? e.g. docker?

**Reply by forestial** [reply]
It turned out that I had configured a token for the config parameter 'auth_token' but then had incorrectly entered the token when the web page first prompted me for it.  So it was simply user error...

Thanks for the reply.
